src/Uno.UI/Mixins/DependencyPropertyMixins.tt

```diff
@@ -6,8 +6,8 @@
 		.Namespace("Windows.UI.Xaml.Controls")
 
 			.Class("Control")
-				.Property("HorizontalContentAlignment", "HorizontalAlignment", "(Uno.UI.FeatureConfiguration.Control.UseLegacyContentAlignment ? HorizontalAlignment.Left : HorizontalAlignment.Center)")
-				.Property("VerticalContentAlignment", "VerticalAlignment", "(Uno.UI.FeatureConfiguration.Control.UseLegacyContentAlignment ? VerticalAlignment.Top : VerticalAlignment.Center)")				
+				.Property("HorizontalContentAlignment", "HorizontalAlignment", "HorizontalAlignment.Left")
```

Comment:
Apparently this flag (`Uno.UI.FeatureConfiguration.Control.UseLegacyContentAlignment`) is used only here, you should either (by order of personal preferences):
* document it as useless / deprecated
* raise an exception if set to `true`
* flag it as deprecated
* remove it (breaking change alert!)